tests/hashing/subtests/avoid-cascade/rollup/rollup.config.js

```diff
@@ -32,7 +32,9 @@ export default {
             fileName: name + '.html',
             source: `<!DOCTYPE html><h1>${name} page</h1>
   <script src="https://unpkg.com/systemjs@6.2.5/dist/system.min.js"></script>
-  <script type="systemjs-importmap">${bundle['import-map.json'].source}</script>
+  <script type="systemjs-importmap" src="${Object.keys(bundle).find(file =>
+    file.startsWith('import-map'),
+  )}"></script>
```

Comment:
This cases an additional roundtrip, but this isn't relevant for this example. Roundtrips doesn't seem to be relevant anywere in `tooling.report`, not sure why?

Reply:
Seems like the kind of thing we'd be interested in. Please propose a test!

Reply:
Might be dupe of https://github.com/GoogleChromeLabs/tooling.report/issues/285?

Reply:
Yep it's reflected in this test